4.gene_expression_signatures/scripts/0.latent_dimension_dataframe.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[1]:
```

Comment:
Can you add a bit more details to a markdown cell above describing what this notebook accomplishes? I assume that it is generating the latent dimension dataframe that you're using to visualize? My assumptions can be confirmed also by renaming the notebook file - I often find that verbs are good for notebook names! (e.g., `0.generate_latent_dimension_dataframe,ipynb`)

Reply:
Separated out and renamed!